rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -26,7 +26,18 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
     def packerCommand = [baseCommand, "packer", "build", "-color=false"]
     parameterMap.each { key, value ->
       if (key && value) {
-        def keyValuePair = value instanceof String && value.contains(" ") ? "$key=\"$value\"" : "$key=$value"
+        def cleansedValue = value instanceof String ? value.trim() : value
```

Comment:
@duftler I wasn't able to track down where the extra quotes were coming from, but haven't looked through the entire Spinnaker codebase yet to figure out how all of it fits together. I originally tried not trimming the quotes, but that didn't work hence my solution. If this is a blocking issue from getting the PR merged, I'll dig in some more this weekend and see if I can track it down. If it's not a blocking issue, I'll add it to my list of Spinnaker related items and will get to it in the next couple of weeks.